tests/bool/good/test_bool.catala_en

```diff
@@ -25,7 +25,7 @@ $ catala Typecheck --check-invariants
 
 ```catala-test-cli
 $ catala Dcalc 
-let TestBool : TestBool_in → TestBool =
+let TestBool : #[test] #[test] TestBool_in → #[test] TestBool =
```

Comment:
Should the printer print attributes like this?

Reply:
I believe that attributes should not be print by default by the printers, they should only come ot with the `--debug` option no?